templates/miq-template-ext-db.yaml

```diff
@@ -265,7 +265,7 @@ objects:
   spec:
     ports:
       -
-        name: "postgresql"
+        name: "postgres"
```

Comment:
![image](https://cloud.githubusercontent.com/assets/651659/26010882/b2926cf8-371c-11e7-856b-f15ccc8c3839.png)
Just because that's the name of the official one, but either is Fine by me.